DataEngineering/fetch_data.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+import pandas as pd
+import json
+import requests
+import mysql.connector as mysql
+
+url = "https://api.tfl.gov.uk/bikepoint"
+response = requests.get(url)
+data = pd.DataFrame(json.loads(response.text))
+
+data['time'] = datetime.now()
+data.drop(['url','$type','placeType','children','childrenUrls'],axis=1,inplace=True)
+
+def getdata(li):
+    for ele in li:
+        if ele["key"] == "NbBikes":
+            data["NbBikes"] = ele["value"]
```

Comment:
This will set the value of the entire column "NbBikes" to whatever value comes into the element. This way, the no of bikes and docks for the last docking station would get set for every Bikepoint.